### DISCUSSION TITLE
exploring-the-dotnet-8-preview-comparing-createbuilder-to-the-new-createslimbuilder-method/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#383] [Announcements]
# exploring-the-dotnet-8-preview-comparing-createbuilder-to-the-new-createslimbuilder-method/

In this post I look at the new WebApplication.CreateSlimBuilder() method, explore why it was introduced, and discuss how it differs from CreateBuilder().

https://andrewlock.net/exploring-the-dotnet-8-preview-comparing-createbuilder-to-the-new-createslimbuilder-method/

**Comment by KennethHoff** [comment]
What about the newly added `CreateEmptyBuilder`? 

https://github.com/dotnet/aspnetcore/pull/49246

Let's hope they actually make it empty this time, so there's no `CreateActuallyEmptyBuilder`

**Reply by largeprob** [reply]
？？

**Comment by burkenyo** [comment]
Kudos! This article is linked from the official ASP.Net Core docs for AOT.
